videointelligence/snippets/object_tracking_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestObjectTracking(t *testing.T) {
+	testutil.SystemTest(t)
+
+	var buf bytes.Buffer
+	err := objectTracking(&buf, "../resources/cat.mp4")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if got := buf.String(); !strings.Contains(got, "cat") {
+		t.Fatalf(`objectTracking("../resources/cat.mp4") = %q; want "cat"`, got)
+	}
+}
+
+func TestObjectTrackingGcs(t *testing.T) {
+	testutil.SystemTest(t)
+
+	var buf bytes.Buffer
+	err := objectTrackingGcs(&buf, "gs://demomaker/cat.mp4")
```

Comment:
Is this publicly available and will be around for a long time? Should probably use the `cloud-devrel-public-resources` project.

Reply:
This is where the canonical Python sample points to, so I'm assuming it will be fine.